seed/views/v3/organizations.py

```diff
@@ -1310,3 +1322,25 @@ def geocoding_enabled(self, request, pk=None):
         org = Organization.objects.get(id=pk)
 
         return org.geocoding_enabled
+
+    @has_perm_class('requires_member')
+    @ajax_request_class
+    @action(detail=True, methods=['GET'])
+    def property_display_field(self, request, pk=None):
+        """
+        Returns the organization's property_display_field setting
+        """
+        org = Organization.objects.get(id=pk)
+
+        return org.property_display_field
+
+    @has_perm_class('requires_member')
+    @ajax_request_class
+    @action(detail=True, methods=['GET'])
+    def taxlot_display_field(self, request, pk=None):
+        """
+        Returns the organization's taxlot_display_field setting
+        """
+        org = Organization.objects.get(id=pk)
```

Comment:
Are these being used anywhere? I didn't see it anywhere, but could have missed it. Wondering if these endpoints are necessary

Reply:
No, they are not being used, I just assumed that new members needed to be added.  I honestly don't know if they will be super usefull either so I can remove them.